Libraries/LibWeb/CSS/Parser/PropertyParsing.cpp

```diff
@@ -839,7 +839,7 @@ RefPtr<CSSStyleValue> Parser::parse_color_scheme_value(TokenStream<ComponentValu
 
         // The 'normal', 'light', 'dark', and 'only' keywords are not valid <custom-ident>s in this property.
         // Note: only 'normal' is blacklisted here because 'light' and 'dark' aren't parsed differently and 'only' is checked for afterwards
-        auto ident = parse_custom_ident_value(tokens, { "normal"sv });
+        auto ident = parse_custom_ident_value(tokens, Array { "normal"sv });
```

Comment:
If you want to save some typing, you could say:
```c++
auto ident = parse_custom_ident_value(tokens, { { "normal"sv } });
```
This hits the `T[Size]` span constructor.